### ISSUE TITLE
Make button opening the JavaScript date picker accessible

### ISSUE THREAD

**Issue by avernet** [orbeon/orbeon-forms#6863]
When the button get the focus, the screenreader should be able to tell the user what the button is about.

[+1 from customer](https://3.basecamp.com/3600924/buckets/6794635/messages/8447453725)
